sites/geohub/src/routes/api/stac/[id]/[collection]/[...item]/tools/[tool]/+server.ts

```diff
@@ -0,0 +1,14 @@
+import type { RequestHandler } from '@sveltejs/kit';
+import { createDatasetLinks } from '$lib/server/helpers';
+import { env } from '$env/dynamic/private';
+
+export const POST: RequestHandler = async ({ request, url, params }) => {
+	const feature = await request.json();
+
+	const tool = params.tool;
+
+	feature.properties = await createDatasetLinks(feature, url.origin, env.TITILER_ENDPOINT, tool);
```

Comment:
do we need this endpoint?

I think `algorithmId` can be included inside feature object as tag. you can get algorithmId from tags.